README.rst

```diff
@@ -20,14 +20,15 @@ Install and update using `pip`_:
 
 .. code-block:: text
 
-    pip install -U Flask
+    $ pip install -U Flask
 
 
 A Simple Example
 ----------------
 
 .. code-block:: python
 
+    # app.py
```

Comment:
Maybe worth adding something to this comment to make it more explicit that this is required? Or maybe just something like `# put this in a file named app.py`?

Reply:
Good point. How about using `# save this as app.py`?

Reply:
Updated. Since you and David agreed, now I'm going to merge it.